app/containers/AskQuestion/index.js

```diff
@@ -0,0 +1,109 @@
+/**
+ *
+ * AskQuestion
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Helmet } from 'react-helmet';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { translationMessages } from 'i18n';
+import { addToast } from 'containers/Toast/actions';
+
+import injectSaga from 'utils/injectSaga';
+import injectReducer from 'utils/injectReducer';
+
+import { makeSelectAccount } from 'containers/AccountProvider/selectors';
+import { makeSelectLocale } from 'containers/LanguageProvider/selectors';
+
+import { askQuestion } from './actions';
+import * as askQuestionSelector from './selectors';
+import reducer from './reducer';
+import saga from './saga';
+import messages from './messages';
+
+import { FORM_TITLE } from './constants';
+
+import AskQuestionForm from './AskQuestionForm';
+
+/* eslint-disable react/prefer-stateless-function */
+export class AskQuestion extends React.Component {
+  postQuestion = values => {
+    const question = {
+      title: values.get(FORM_TITLE),
+      content: window.tinyMCE.activeEditor.getContent(),
```

Comment:
All code related to tinyMCE has to be encapsulated inside TexEditor component. If we need to replace TinyMCE with something else in the future then TextEditor should be the only component that requires changes. 
Components using TextEditor component should no know nothing about TinyMCE being used inside of it.